### ISSUE TITLE
voice input completely unusable with talkback enabled in Android 4.4 kitkat

### ISSUE THREAD

**Issue by GoogleCodeExporter** [clc/eyes-free#333]
```
What steps will reproduce the problem?
1. acivate talkback
2. activate voice input in any text input field
3. speak a text like "this is a test"

What is the expected output? What do you see instead?
voice input would produce "this is a text" and talkback would repeat it after 
ending voice-input.
instead talkback repeats every word DURING the voice input, so voice input does 
recognize talkback's voice as new input. Produces something like "this is a 
test this is a test is a test text texting [...]"

What version of the product are you using? On what operating system?
Andoid 4.4 KitKat KRT16S on Nexus 4 and Nexus 7 (2013)
Talkback 3.5.0
Google text-to-speech 2.4.3.920375
Google keyboard 2.0.19003.920375a
Google search 3.1.8.914827.arm

Please provide any additional information below.
worked before updating android to 4.4 from 4.3

```

Original issue reported on code.google.com by `m...@nexero.de` on 1 Dec 2013 at 9:50

**Comment by GoogleCodeExporter** [comment]
Original comment by `caseybur...@google.com` on 5 Dec 2013 at 6:09
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 3.5.1
```

Original comment by `caseybur...@google.com` on 6 Dec 2013 at 3:28
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Talkback 3.5.1 beta fixed the voice input bug partly.
In Gmail it works perfect but in Quickoffice 6.2.5.310b the text is recognized 
and typed but the moment you end voice input the entire input gets deleted. 

Using a Nexus 7 (2013) with Android 4.4 Kitkat
```

Original comment by `hdbr...@gmail.com` on 9 Dec 2013 at 10:54
